tests/neg-custom-args/captures/boxmap.check

```diff
@@ -1,7 +1,7 @@
 -- [E007] Type Mismatch Error: tests/neg-custom-args/captures/boxmap.scala:12:2 ----------------------------------------
```

Comment:
Since this checkfile's source was deleted, should it be deleted as well?

Reply:
Ah yes, that was an oversight

Reply:
Do you (or someone else) want to give this a quick review? But I am not sure the PR makes much sense when seen in isolation. I planned as next step to merge this and then make a PR of the whole cc-experiment branch against main. That would need a thorough review.